src/kernel/src/fs/dev/cdev.rs

```diff
@@ -237,5 +233,5 @@ pub(super) enum DefaultError {
 
     #[error("ioctl not supported")]
     #[errno(ENODEV)]
-    IoctlNotSupported,
+    CommandNotSupported,
```

Comment:
`IoctlNotSupported` looks better than the new version in this case because `IoctlNotSupported` means the method is not supported regardless the `cmd` passed in.